packages/next/src/data/server/addHookData.ts

```diff
@@ -32,6 +32,19 @@ function isAppEntity(data: Entity): data is AppEntity {
 	return typeof (data as AppEntity).settings !== 'undefined';
 }
 
+export interface AddHokoDataBaseProps {
+	[key: string]: unknown;
+}
```

Comment:
You may want to be careful with allowing wildcard props, unless you're explicitly planning to access them.

Reply:
`addHookData` should be able to receive `props` that would then be passed to the page component in Next.js. This can be arbitrary props. This was mostly to allow TS to accept arbritary props and to infer the type from the passed props as well.